pkgs/development/misc/resholve/oildev.nix

```diff
@@ -58,59 +38,54 @@ rec {
         hash on rev updates. Command will fail w/o and not print hash.
       */
       postFetch = ''
-        rm -rf $out/{Python-2.7.13,metrics,py-yajl,rfc,gold,web,testdata,services,demo,devtools}
+        rm -rf $out/{Python-2.7.13,metrics,py-yajl,rfc,gold,web,testdata,services,demo}
```

Comment:
Small nit here, but could we extract the Python version from the package itself?

Alternatively, we could use `Python-*`.

Reply:
This is just oil's vendored python